internal/log/log.go

```diff
@@ -16,7 +16,12 @@
 
 package log
 
-import "sync"
+import (
+	"sync"
+
+	"github.com/vdaas/vald/internal/log/glg"
+	"github.com/vdaas/vald/internal/log/mode"
+)
 
 type Logger interface {
 	Info(vals ...interface{})
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
The order of the interface methods should follow the logging level ordering.